--- libsepol-1.16.0/src/Makefile.orig	2007-03-02 16:18:12.000000000 +0100
+++ libsepol-1.16.0/src/Makefile	2007-03-02 16:18:24.000000000 +0100
@@ -20,7 +20,7 @@
 	ranlib $@
 
 $(LIBSO): $(LOBJS)
-	$(CC) $(LDFLAGS) -shared -o $@ $^ -Wl,-soname,$(LIBSO),--version-script=libsepol.map,-z,defs
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -o $@ $^ -Wl,-soname,$(LIBSO),--version-script=libsepol.map,-z,defs
 	ln -sf $@ $(TARGET) 
 
 %.o:  %.c 
